src/guides/v2.4/graphql/tutorials/checkout/checkout-shipping-method.md

```diff
@@ -22,21 +22,22 @@ The `setShippingMethodsOnCart` mutation defines the delivery methods for your or
 
 `{ CART_ID }` is the unique shopping cart ID from [Step 2. Create empty cart]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-add-product-to-cart.html).
 
-{:.bs-callout .bs-callout-info}
+The `carrier_code` and `method_code` values come from the results of the mutation on the previous step.
```

Comment:
It's currently missing from the tutorial. This should link to the shipping address step

Reply:
Done